src/boardlib/api/aurora.py

```diff
@@ -473,56 +492,87 @@ def combine_ascents_and_bids(ascents_df, bids_summary, db_path, grades_dict, gra
+                    'date': ascent_row['date'],
+                    'tries': total_tries
+                }
+            else:
+                entry = {
+                    'board': ascent_row['board'],
+                    'angle': ascent_row['angle'],
+                    'climb_name': ascent_row['name'],
+                    'date': ascent_row['date'],
+                    'logged_grade': ascent_row['logged_grade'],
+                    'displayed_grade': ascent_row.get('displayed_grade', None),
+                    'difficulty': ascent_row['difficulty'],
+                    'tries': total_tries,
+                    'is_mirror': ascent_row['is_mirror'],
+                    'is_ascent': True,
+                    'comment': ascent_row['comment']
+                }
+            final_logbook.append(entry)
             bids_summary = bids_summary.drop(bid_match.index)
         else:
-            final_logbook.append({
-                'board': ascent_row['board'],
-                'angle': ascent_row['angle'],
-                'climb_name': ascent_row['name'],
-                'date': ascent_row['date'],
-                'logged_grade': ascent_row['logged_grade'],
-                'displayed_grade': ascent_row.get('displayed_grade', None),
-                'difficulty': ascent_row['difficulty'],
-                'tries': ascent_row['tries'],
-                'is_mirror': ascent_row['is_mirror'],
-                'is_ascent': True,
-                'comment': ascent_row['comment']
-            })
+            if aggregate:
+                entry = {
+                    'uid': f"{ascent_row['climb_uuid']}-{ascent_row['angle']}",
+                    'board': ascent_row['board'],
+                    'climb_name': ascent_row['name'],
+                    'date': ascent_row['date'],
+                    'tries': ascent_row['tries']
+                }
+            else:
+                entry = {
+                    'board': ascent_row['board'],
+                    'angle': ascent_row['angle'],
+                    'climb_name': ascent_row['name'],
+                    'date': ascent_row['date'],
+                    'logged_grade': ascent_row['logged_grade'],
+                    'displayed_grade': ascent_row.get('displayed_grade', None),
+                    'difficulty': ascent_row['difficulty'],
+                    'tries': ascent_row['tries'],
+                    'is_mirror': ascent_row['is_mirror'],
+                    'is_ascent': True,
+                    'comment': ascent_row['comment']
+                }
+            final_logbook.append(entry)
 
     for _, bid_row in bids_summary.iterrows():
-        if db_path:
-            difficulty = get_difficulty_from_db(db_path, bid_row["climb_uuid"], bid_row["angle"])
-            displayed_grade = convert_difficulty_to_grade(difficulty, grades_dict, grade_type)
+        if aggregate:
+            entry = {
+                'uid': f"{bid_row['climb_uuid']}-{bid_row['angle']}",
```

Comment:
Let's stay consistent with `uuid`, confusing to drop the `u` here. If you want, it could also be `climb_angle_uuid` to be more descriptive. Maybe I'm missing something here, but couldn't Climbdex construct the climb + angle ID on that end so that boardlib didn't need to worry about this nuance?